.github/workflows/tests.yml

```diff
@@ -259,6 +259,7 @@ jobs:
           -e TEST_GROUP
           ghcr.io/conda/conda-ci:main-linux-python${{ matrix.python-version }}${{ matrix.default-channel == 'conda-forge' && '-conda-forge' || '' }}
           bash -c "source /opt/conda/etc/profile.d/conda.sh; \
+                   sudo /opt/conda/bin/conda install --quiet -y -c defaults --file tests/requirements.txt; \
```

Comment:
I'm surprised this hasn't been merged already but we have to make sure our new test dependencies (flask) are installed, and the cached docker image might not have those.